src/features/canvass/components/LocationDialog/index.tsx

```diff
@@ -148,6 +158,18 @@ const LocationDialog: FC<LocationDialogProps> = ({
               location={location}
               onBack={() => back()}
               onClose={onClose}
+              onDelete={() => {
+                showConfirmDialog({
+                  disablePortal: true,
+                  onSubmit: () => {
+                    deleteHousehold(selectedHouseholdId);
```

Comment:
This can fail, if the user doesn't have the right permissions. Maybe now is not the right time to resolve this, but we should handle the error somehow eventually.

Reply:
I’ve noted it down to address it in the future.